TAA/TAA.md

```diff
@@ -0,0 +1,141 @@
+# Transaction Author Agreement V2
+```
+                             `-::-`                      
+                      ```   `//////`   ```               
+                    .////:` `//////` `:////.             
+                    //////-  `-::-`  -//////             
+                    .:////`    --    `////:.             
+                ....   ` .:`   --   `:. `   ....         
+              `/////:     `:.  ..  .:`     :////:`       
+              .//////-.`    .  ``  .    `.-//////.       
+               .:::````.---  -////-  ---.```::::.        
+                            .//////.                     
+               .-::-` `.---  -////-  ---.` `-::-.        
+              .//////-.`    `  ``  .    `.-//////.       
+              `/////:`    `:.  ..  .:`    `://///`       
+                .--.     .:`   --   `:.     .--.         
+                    .:////`    --    `////:.             
+                    //////-  `.::.`  -//////             
+                    .////:` `//////` `:////.             
+                      ```   `//////`   ```               
+                             `-::-`                      
+                                                                    
+ ssSSSSSss                                         II               
+sS"     "Ss                                        ""               
+SS,                                                                 
+`SSsssss,     ,ooOOOoo,   Vv       vV  Rr,rRRRrr,  II  Nn,nNNNnn,   
+  `"""""Ss,  oO"     "Oo  `Vv     vV'  RRR'   "RR  II  NNN'   `"Nn  
+        `Ss  OO       OO   `Vv   vv'   RR          II  NN       NN  
+SSs     SSS  "Oo,   ,oO"    `Vv,vV'    RR          II  NN       NN  
+ "SSSSSSS"    `"OoooO"'       "V"      RR          II  NN       NN  
```

Comment:
While the ASCII art is pretty cool, maybe we shouldn't write it to the ledger lol

Reply:
the original document has a logo. maybe we should add `its time to exist` or `identity for all` to better match the original text?

Reply:
I will look for official approval.